src/overlays/actors/ovl_Demo_Tre_Lgt/z_demo_tre_lgt.h

```diff
@@ -6,11 +6,33 @@
 
 struct DemoTreLgt;
 
+typedef void (*DemoTreLgtActionFunc)(struct DemoTreLgt*, GlobalContext*);
+
+typedef struct {
+    /* 0x00 */ f32 startFrame;
+    /* 0x04 */ f32 endFrame;
+    /* 0x08 */ f32 unk_08;
+    /* 0x0C */ f32 unk_0C;
+} FrameInfo; // size = 0x10
+
+typedef struct {
+    /* 0x00 */ FrameInfo frameInfo[2];
+} DemoTreLgtInfo; // size = 0x20
```

Comment:
Any reason not to make` sDemoTreLgtInfo` an array of `FrameInfo` instead of making an additional struct for it? Also, `FrameInfo` should probably be moved to the C file, or renamed to something more specific to avoid conflicts.

Reply:
It wouldn't match unless I made them separate structs

Reply:
That shouldn't ever matter afaik

Reply:
you were right, idk why i thought it only matched like that. maybe i was trying to match something and changed that but also changed something else and it made me think it was that